### ISSUE TITLE
Title bar text is hard to read

### ISSUE THREAD

**Issue by ray-x** [nvim-neo-tree/neo-tree.nvim#753]
With some of popular colorschemes (e.g. catppuccin) the Bar area of neotree is still hard to read without additional config/setup.
Maybe change it and link to `PmenuSel` or `PmenuSbar` where even the default theme from nvim can handle it easily. 
<img width="265" alt="image" src="https://user-images.githubusercontent.com/1681295/219230469-301ad259-2f35-44d2-8bf1-781a15ecb0c3.png">

<img width="265" alt="image" src="https://user-images.githubusercontent.com/1681295/219230661-9dea7132-9611-4242-8f3e-fe4af54b5856.png">

<img width="265" alt="image" src="https://user-images.githubusercontent.com/1681295/219231237-4194bbd1-8e3a-43f5-b1f2-0c4df5425ee9.png">

<img width="265" alt="image" src="https://user-images.githubusercontent.com/1681295/219234010-67e2dde4-aabc-4b4c-b08d-bec9cd573e04.png">

**Comment by cseickel** [comment]
I feel like this is something that just can't be helped. Whatever choice we make is going to be bad for someone.

Maybe a better solution is to stop using such a customized border and just go with the standard "rounded" look by default, which tends to work out better with most themes.

Another fix is to just come up with good highlight choices for popular themes and submit them for inclusion in those projects.

**Comment by ray-x** [comment]
This also works for me ` stop using such a customized border and just go with the standard "rounded" look by default`
In fact, the default dressing round border looks better from my perspective:

![image](https://user-images.githubusercontent.com/1681295/219542311-07520d4b-a2cb-4987-9149-6a2489310655.png)
and the nui default one:
https://github.com/MunifTanjim/nui.nvim#input

**Comment by miversen33** [comment]
@cseickel thoughts on having an an optional dependency on Dressing? If dressing is available, (or maybe allowing a configuration option to be passed to say `use_plain_popup_borders`?) we use that instead? I agree with @ray-x , I think the dressing popup looks better but I don't think adding a hard dependency to Neo-tree is the right option.

**Comment by cseickel** [comment]
@miversen33 @ray-x Sorry I wasn't clear. There is nothing to do here other than change the default config option in the next major release. This is actually an existing config option:

```lua
require("neo-tree").setup({
  popup_border_style = "rounded"
})
```

You can also disable the built-in popups entirely, and if you have enabled another library as the `vim.ui.input` handler it will be used:
use_popups_for_input = true, -- If false, inputs will use vim.ui.input() instead of custom floats.

**Comment by miversen33** [comment]
That sounds right. We should probably document that somewhere, I don't believe its in our readme or the help docs. I imagine we should wholesale give the docs a bit of love when it comes to configuration options

**Comment by cseickel** [comment]
> I imagine we should wholesale give the docs a bit of love when it comes to configuration options

Absolutely. It's sorely needed but I hate writing documentation. If I could magically get help with one thing on this project, it would be that.

**Comment by miversen33** [comment]
I feel this. When I see something that I know isn't documented (and I generally know what its doing), I'll provide a bit of help there :) This is a reasonable example. I will see if I can get this updated tonight when I get home

**Comment by miversen33** [comment]
It seems that only some of the popup related things respect this option currently. Example `fuzzy_finder` still opens the neo-tree default popup if you have `use_popups_for_input = false` whereas `add` respects the setting. This is not an issue for the `rounded` setting you mentioned earlier, but it does seem that the popups will ignore the `use_popups_for_input` config flag at times. I have not dug into the code to figure out why this is (my guess is that fuzzy_finder simply bypasses the ui config but :man_shrugging: )

**Comment by cseickel** [comment]
> `fuzzy_finder` still opens the neo-tree default popup if you have `use_popups_for_input = false`

Oh yeah, I remember trying to make that work with the standard vim input but I found it was impossible to react to key presses with it being submitted. I'm not sure if dressing could work or not for that use case.

**Comment by pysan3** [comment]
Hi @miversen33 , just pinging you as a reminder :)

**Comment by miversen33** [comment]
I appreciate you :) I need to get all the work I volunteered for organized lol. This seems pretty low priority so I will get to this "eventually", unless someone else wants to take a crack at it
